pdc/apps/release/models.py

```diff
@@ -236,6 +234,19 @@ def version_sort_key(self):
         except ValueError:
             return [self.short]
 
+    @property
+    def integrated_release_variants(self):
+        """
+        Returns mapping from Variant-UID to release instance from which the
+        variant is integrated.
+        """
+        if not hasattr(self, '_integrated_release_variants'):
+            self._integrated_release_variants = {}
+            for release in self.integrated_releases.all():
```

Comment:
Should here also use prefetch_related to reduce DB query number, something like:
for release in self.integrated_releases.prefetch_related('variant_set')?